publish.Dockerfile

```
@@ -0,0 +1,47 @@
+
```

```suggestion
ARG DAED_VERSION
```

We pass the version from CI as `build-arg`